src/layouts/base-new/index.tsx

```diff
@@ -1,27 +1,13 @@
-import AlertBanner from 'components/alert-banner'
+import DevAlertBanner from 'components/dev-alert-banner'
 import NavigationHeader from 'components/navigation-header'
-import s from './base-new-layout.module.css'
 
 const BaseNewLayout: React.FC = ({ children }) => {
   return (
-    <div className={s.baseNewLayout}>
-      <AlertBanner type="highlight">
-        <p>
-          You are viewing an internal preview and work in progress version of
-          this site.{' '}
-          <a
-            href="https://airtable.com/shrU3eYHIOXO60o23"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            We&apos;d love to hear your feedback
-          </a>
-          !
-        </p>
-      </AlertBanner>
+    <>
+      <DevAlertBanner />
```

Comment:
Abstracted `DevAlertBanner` so it could be re-used in `DocsLayout` without having to use this layout as the underlying layout.